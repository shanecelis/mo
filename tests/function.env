name=Willy
wrapped() {
    # The final newline is eaten by mo
    echo "<b>$1</b>"
}
