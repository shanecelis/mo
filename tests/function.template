{{#wrapped}}
  {{name}} is awesome.
{{/wrapped}}
... this is the last line.
