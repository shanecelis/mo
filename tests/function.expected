<b>  Willy is awesome.
</b>... this is the last line.
